docs/runbooks/CDIDefaultStorageClassDegraded.md

```diff
@@ -0,0 +1,47 @@
+# CDIDefaultStorageClassDegraded
+<!-- Edited by agilboa, 4 Dec 2023 -->
+
+## Meaning
+
+This alert fires when the default (Kubernetes or virtualization) storage class supports smart clone (either CSI or snapshot based) and ReadWriteMany.
+
+A default virtualization storage class has precedence over a default Kubernetes storage class for creating a VirtualMachine disk image.
+
+## Impact
+
+If the default storage class does not support smart clone, we fallback to host-assisted cloning, which is the least efficient method of cloning.
+
+If the default storage class does not suppprt ReadWriteMany, a virtual machine using it is not live-migratable.
+
+## Diagnosis
+
+Get the default Kubernetes storage class by running the following command:
+```bash
+$ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubernetes.io/is-default-class"=="true")|.name'
+```
+
+Get the default virtualization storage class by:
+```bash
+$ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
+```
+
+Check the default storage class supports ReadWriteMany:
+```bash
+$ kubectl get storageprofile <default-storage-class-name> -o json | jq '.status.claimPropertySets'| grep ReadWriteMany
```

Comment:
@arnongilboa Is the <default-storage-class-name>  the "virtualization storage class" or "Kubernetes storage class"?

Reply:
fixed